payload-processing/payload_processing.h

```diff
@@ -0,0 +1,10 @@
+/*  payload_processing.h
+
+	Created by Tyrel Kostyk on February 11th 2020
+ */
+
+
+#define DOSIMETER_REFERENCE_VOLTAGE	(float) 3.30;
```

Comment:
```suggestion
#define DOSIMETER_REFERENCE_VOLTAGE	( (float) 3.30 )
```
Macros don't need semi colons. I would also wrap it in brackets just to be safe

Reply:
Also should probably add the documentation comments in if you have time. Other than that the code looks good to me